internal/packmgr/registry.go

```
@@ -0,0 +1,331 @@
+		if err != nil {
+			return "", "", err
+		}
+		if c.Token != "" {
+			request.Header.Set("Authorization", "Bearer "+c.Token)
+		}
+		response, err := c.HTTP.Do(request)
+		if err != nil {
+			return "", "", err
+		}
+		if response.StatusCode == http.StatusNotFound {
+			response.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `response.Body.Close` is not checked

(errcheck)